_build.ham

```diff
@@ -0,0 +1,16 @@
+SubDir TOP ;
+
+local BIN_DIR = [ FDirName $(TOP) bin ] ;
```

Comment:
$(TOP) should be $(SUBDIR) here. The SubDir statement at the top defines SUBDIR and is better for maintenance.